src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/external/EmailDomainController.kt

```diff
@@ -178,7 +179,16 @@ data class EmailDomainDto(
   val domain: String,
   @Schema(required = false, description = "Email domain description", example = "CAREUK")
   val description: String
-)
+) {
+  companion object {
+    fun fromDomain(emailDomain: EmailDomain) =
+      EmailDomainDto(
+        emailDomain.id,
+        emailDomain.domain,
+        emailDomain.description
+      )
```

Comment:
I would probably just do this as an extra constructor

Reply:
Discussed with @karenmillermoj and agreed OK